geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/conf/GeoMesaSystemProperties.scala

```diff
@@ -45,8 +45,17 @@ object GeoMesaSystemProperties extends LazyLogging {
       }
     }
 
+    def toInt: Option[Int] = option.flatMap { value =>
+      Try(java.lang.Integer.parseInt(value)) match {
+        case Success(v) => Some(v)
+        case Failure(_) =>
+          logger.warn(s"Invalid Int for property $property: $value")
+          Option(default).map(java.lang.Integer.parseInt)
+      }
+    }
+
     def toBoolean: Option[Boolean] = option.flatMap { value =>
-      Try(value.toBoolean) match {
+      Try(java.lang.Boolean.parseBoolean(value)) match {
```

Comment:
For my education, what's the difference between these two calls?

Reply:
I think i was mainly trying to match the syntax of the call to `Integer.parseInt` above... looking into it, there is actually a difference, in that `parseBoolean` will always return a Boolean, while `.toBoolean` will throw an exception if it's not either (case-insensitive) 'true' or 'false', which actually seems better. I'm going to switch it back